src/main/java/com/example/tournamentMaker/team/FootballPlayerRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.example.tournamentMaker.team;
+
+import com.example.tournamentMaker.team.player.FootballPosition;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+class FootballPlayerRequest {
+    private String teamName;
+    private String firstName;
+    private String lastName;
+    private Integer jerseyNumber;
+    private FootballPosition footballPosition;
+
+    @JsonProperty("FootballPosition")
+    public void setFootballPosition(String position) {
+        this.footballPosition = FootballPosition.valueOf(position);
+    }
+}
```

Comment:
moze uda sie to uogolnic, bo bedzie trzeba dorobic kolejny sport i moze uda sie zrobic ogolny playerRequest i potem dziedziczyc